src/item.cpp

```diff
@@ -315,7 +315,7 @@ item::item( const recipe *rec, int qty, std::list<item> items, std::vector<item_
 }
 
 item item::make_corpse( const mtype_id &mt, time_point turn, const std::string &name,
-                        const int upgrade_time )
```

Comment:
This const is not useless. The compiler complains about the const in the *declaration* of that function. There it is useless indeed (and ignored by the compiler).

This is because it has no meaning to the caller: a function `foo( int )` is exactly the same as `foo( const int )`. The caller has to copy the `int` in either case, so it does not matter whether the function considers it const or not.

Reply:
Oops removed the wrong one U_U